javascript/src/test/pages/Messages/SearchResults.test.js

```diff
@@ -4,45 +4,14 @@ import { useParams} from "react-router-dom";
 import SearchResults from "main/pages/Messages/SearchResults";
 import userEvent from "@testing-library/user-event";
 
-import useSWR from "swr";
-jest.mock("swr");
-jest.mock("react-router-dom", () => {
-    return {
-        'useParams': jest.fn(),
-    };
-});
-
-describe("ChannelPageList tests", () => {
-
+describe("ChannelPage tests", () => {
     test("renders without crashing", () => {
-        useSWR.mockReturnValue({'data': []});
         render(<SearchResults />);
     });
 
     test("renders without crashing", () => {
-        useSWR.mockReturnValue({'data': []});
         const { getByLabelText } = render(<SearchResults />);
         const selectSearchString = getByLabelText("Search String");
         userEvent.type(selectSearchString, "springboot");
     });
-
-    test("loads messages from the backend", () => {
-        const exampleMessage = {
-            "type": "message",
-            "subtype": "channel_join",
-            "ts": "1594143066.000200",
-            "user": "U017218J9B3",
-            "text": "Someone has joined the channel",
-            "channel": "section-6pm"
-        }
-
-
-        useSWR.mockReturnValue({
-            'data': [exampleMessage]
-        });
-
-        const { getByText } = render(<SearchResults />);
-        const contentsElement = getByText(exampleMessage.text);
-        expect(contentsElement).toBeInTheDocument();
-    });

```

Comment:
The file name is fine here, but we're still removing tests that were created by another team. Also, in line 7, test description is still listed as "ChannelPage tests".